.travis/runtox

```diff
@@ -1,16 +1,44 @@
 #!/bin/bash
 # SPDX-License-Identifier: MIT
 
-set -ex
+# Run tox. Additionally, if LSR_MSCENARIOS is defined, run `tox -e molecule`
+# for every scenario from LSR_MSCENARIOS and for every Ansible version from
+# LSR_ANSIBLES.
+#
+# LSR_MSCENARIOS is a space separated list of molecule scenarios.
+# LSR_ANSIBLES is a space separated list of Ansible package names with versions
+# in pip format, i.e 'ansible ansible==2.6 ansible==2.7 ansible=2.8'.
+#
+# LSR_MSCENARIOS and LSR_ANSIBLES can be set in .travis/config.sh or as
+# environment variables.
 
-SCRIPTDIR=$(dirname $0)
-CONFIG=${SCRIPTDIR}/config.sh
+ME=$(basename $0)
+SCRIPTDIR=$(readlink -f $(dirname $0))
+BANNERSIZE=90
 
-if [[ -f ${CONFIG} ]]; then
-  . ${CONFIG}
+. ${SCRIPTDIR}/utils.sh
+. ${SCRIPTDIR}/config.sh
+
+lsr_banner "tox" ${BANNERSIZE}
+(set -x; tox); error_code=$?
+
+if lsr_check_python_version python -le '3.4'; then
```

Comment:
This now tries to run molecule on Travis for all Python versions but then skips the run because the system libraries can only be accessed for 3.5. This adds a lot of noise to the logs. Technically it should be possible to run all molecule tests except for the molecule_test version with other Python versions (IIRC). Only the molecule_test actually uses the container image. But we probably do not want to do this, running it once is enough, isn't it?

Therefore I propose to also add the check for the python version here to reduce the noise. Or is this not possible because this is before tox?

Reply:
Thanks for noting this. Actually, as Travis documentation says, `python` command here refers also to virtualenv Python, so it should be no problem to implement the check as:
```bash
if ! lsr_venv_python_matches_system_python; then
  exit $error_code
fi
```